runtime/src/bank.rs

```diff
@@ -5882,6 +5878,13 @@ impl Bank {
         Some(vote_account.clone())
     }
 
+    /// Get the EpochStakes for the current Bank::epoch
+    pub fn current_epoch_stakes(&self) -> &EpochStakes {
+        self.epoch_stakes
+            .get(&self.epoch)
```

Comment:
I _think_ the closest thing to the old `staked_nodes` is `self.epoch_stakes[self.epoch + 1]`.

Reply:
not a fan of "leader epoch" but I guess whatever we name it would be pretty ambiguous. maybe ask in discord if people have any suggestions.

Reply:
The stakes for a given epoch (`E`) in `self.epoch_stakes` are keyed by leader schedule epoch (`E + 1`) so if you want the stakes for the current epoch, you need to use `self.epoch_stakes[E + 1]` as @behzadnouri mentioned but the method name `current_epoch_stakes` you've chosen still makes sense to me. In the future we might need to change the naming to distinguish properly between current stake distribution and active stake distribution but no need to worry about that in this PR in my opinion.